lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -166,8 +166,12 @@ def instructor_dashboard_2(request, course_id):
 
     certificate_white_list = CertificateWhitelist.get_certificate_white_list(course_key)
     certificate_exception_url = reverse(
```

Comment:
I think we should rename certificate_exception_url  to `generate_certificate_exceptions_url` to make it more readable and now we have `certificate_exception_view_url ` which makes it ambiguous.